protocol/contracts/oracle/src/error.rs

```diff
@@ -37,6 +37,9 @@ pub enum ContractError {
     #[error("[Oracle] Failed to store configuration! Cause: {0}")]
     StoreConfig(StdError),
 
+    #[error("[Oracle] Failed to load the v0 configuration! Cause: {0}")]
+    LoadConfigV0(StdError),
```

Comment:
[Minor] Adding a `TODO` comment might be good, just to indicate that it's not needed after deployment.